site/src/pages/UserSettingsPage/TokensPage/TokensPageView.tsx

```diff
@@ -0,0 +1,131 @@
+import { useTheme } from "@material-ui/core/styles"
+import Table from "@material-ui/core/Table"
+import TableBody from "@material-ui/core/TableBody"
+import TableCell from "@material-ui/core/TableCell"
+import TableContainer from "@material-ui/core/TableContainer"
+import TableHead from "@material-ui/core/TableHead"
+import TableRow from "@material-ui/core/TableRow"
+import { APIKey } from "api/typesGenerated"
+import { ChooseOne, Cond } from "components/Conditionals/ChooseOne"
+import { Stack } from "components/Stack/Stack"
+import { TableEmpty } from "components/TableEmpty/TableEmpty"
+import { TableLoader } from "components/TableLoader/TableLoader"
+import DeleteOutlineIcon from "@material-ui/icons/DeleteOutline"
+import dayjs from "dayjs"
+import { FC } from "react"
+import { AlertBanner } from "components/AlertBanner/AlertBanner"
+import IconButton from "@material-ui/core/IconButton/IconButton"
+
+export const Language = {
+  idLabel: "ID",
+  createdAtLabel: "Created At",
+  lastUsedLabel: "Last Used",
+  expiresAtLabel: "Expires At",
+  emptyMessage: "No tokens found",
+  ariaDeleteLabel: "Delete Token",
+}
```

Comment:
We have an alternative to `Language` now, react-i18n, but if it's a hassle, don't worry about it, we're going to revisit whether it's worth the time